src/Ledger/Transaction.lagda

```diff
@@ -235,8 +235,12 @@ Ingredients of the transaction body introduced in the Conway era are the followi
 \end{NoConway}
 
 \begin{code}[hide]
-  isP2Script : Script → Bool
-  isP2Script = is-just ∘ isInj₂
+  isP2Script : Script → Type
+  isP2Script = From-inj₂
+
+  isP2Script? : ∀ {s} → isP2Script s ⁇
+  isP2Script? {inj₁ x} .dec = true because ofʸ (lift tt)
+  isP2Script? {inj₂ y} .dec = true because ofʸ y
```

Comment:
You can also see it here, it would be odd if we had a property that's always true. Also, we typically use `yes x` instead of `true because ofʸ x`.